frontend/src/components/landing/Reviews.module.scss

```
@@ -257,14 +252,22 @@
   }
 }
 
-// mobile only controls for review scrolling
+@media (prefers-reduced-motion) {
+  .scroll-right,
+  .scroll-left {
+    animation: none;
+  }
+}
+
+// mobile only controls for review scrollingW
 .mobile-controls {
   display: flex;
-  gap: 8px;
+  justify-content: space-between;
+  gap: $spacing-sm;
 
   button {
-    width: 50%;
-    padding: 16px 0;
+    width: 48%;
```

This is a bit of a magic number. I assume this was to prevent the border from being invisible? If so, it doesn't seem necessary anymore now that `.show-mobile` isn't `inline-block` anymore:

```suggestion
    width: 50%;
```